internal/controller/olsconfig_controller.go

```diff
@@ -80,6 +80,13 @@ type OLSConfigReconcilerOptions struct {
 // +kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=clusterroles;clusterrolebindings,verbs=*
 // +kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=roles;rolebindings,verbs=*
 
+// RBAC for application server to authorize user for API access
+// +kubebuilder:rbac:groups=authorization.k8s.io,resources=subjectaccessreviews,verbs=create
+// +kubebuilder:rbac:groups=authentication.k8s.io,resources=tokenreviews,verbs=create
```

Comment:
@raptorsun do we need delete to clean up , once the operator is uninstalled ?

Reply:
the create verb should be just for allowing "post" HTTP method when sending subjectaccessreviews and tokenreviews to the kube api server

Reply:
@raptorsun - Still checking if we need this create for operator ServiceAccount. This operation is only done using ServiceAccount for Appserver.

Reply:
The operator's service account must have access to `tokenreview` before granting it to app server's service account